docs/source/tutorial.rst

```diff
@@ -34,21 +45,26 @@ is initialized:
         cache_kws=dict(available_bytes=1e9)
     )
 
-Serving a dataset simply requires calling the :meth:`~xarray.Dataset.rest.serve`
-method on the ``rest`` accessor:
+Serving the dataset then simply requires calling the
+:meth:`~xpublish.Rest.serve` method on the ``Rest`` instance or the ``rest``
+accessor:
 
 .. code-block:: python
 
+    rest.serve()
+
+    # or
+
     ds.rest.serve()
 
-:meth:`~xarray.Dataset.rest.serve` passes any keyword arguments on to
+:meth:`~xpublish.Rest.serve` passes any keyword arguments on to
 :func:`uvicorn.run`.
```

Comment:
perhaps link to here: https://www.uvicorn.org/deployment/#running-programmatically ?